nginx.conf

```diff
@@ -7,8 +7,8 @@ http {
             proxy_pass http://frontend:3000;
         }
         # api
-        location /api/ {
-            rewrite /api/(.*) /$1 break;
+        location /backend/ {
+            rewrite /backend/(.*) /$1 break;
```

Comment:
If you change it in nginx, let's also change it in the ingress (I think you will need to wait for Denis or merge fork yourself into this repo first)

Reply:
https://github.com/makerdao-ses/switchboard-boilerplate/pull/42#pullrequestreview-1383084858

addressed according to the link above